mlflow/R/mlflow/.run-tests.R

```diff
@@ -1,15 +1,9 @@
 parent_dir <- dir("../", full.names = TRUE)
 package <- parent_dir[grepl("mlflow_", parent_dir)]
-install.packages(package)
 
-mlflow:::mlflow_maybe_create_conda_env(python_version = "3.6")
+library(mlflow)
 library(reticulate)
```

Comment:
I am not sure it's good practice to load packages other than testthat and the package to test when running tests. Can we remove this and change the below line to `use_condaenv(mlflow:::mlflow_conda_env_name())`?

Reply:
> Can we remove this and change the below line ...

which line?